fsi/fsi.go

```diff
@@ -253,6 +275,70 @@ func removeLinkFile(dir string) error {
 	return os.Remove(dir)
 }
 
+func isLowValueFile(f os.FileInfo) bool {
+	if f != nil {
+		filesToBeRemoved := []string{
+			// generic files
+			"^\\..*\\.swp$", // Swap file for vim state
+
+			// macOS specific files
+			"^\\.DS_Store$", // Stores custom folder attributes
+			"^\\.AppleDouble$", // Stores additional file resources
+			"^\\.LSOverride$", // Contains the absolute path to the app to be used
+			"^Icon\\r$", // Custom Finder icon: http://superuser.com/questions/298785/icon-file-on-os-x-desktop
+			"^\\._.*", // Thumbnail
+			"^\\.Spotlight-V100(?:$|\\/)", // Directory that might appear on external disk
+			"\\.Trashes", // File that might appear on external disk
+			"^__MACOSX$", // Resource fork
+
+			// linux specific files
+			"~$", // Backup file
```

Comment:
Not fan of this one pattern. Any file ending with a `~` will be considered low-value, that's too broad and easy to get false positives.

Reply:
Happy to remove it, mostly picked it up from some other list of "low value files"